cmd/crc/cmd/delete.go

```diff
@@ -34,6 +34,16 @@ func runDelete(arguments []string) {
 	deleteConfig := machine.DeleteConfig{
 		Name: constants.DefaultName,
 	}
+
+	// Check if the crc instance exists
+	exist, err := machine.MachineExists(deleteConfig.Name)
+	if err != nil {
+		errors.ExitWithMessage(1, err.Error())
+	}
+	if !exist {
+		errors.ExitWithMessage(1, machineNotExistsErrorMessage)
+	}
+
```

Comment:
This could go to a helper `checkMachineExists(machineName string)`/`assertMachineExists`/`exitIfMachineMissing` rather than duplicating the same code multiple times

Reply:
@cfergeau make sense, updated :+1: